tensorflow_lite_support/python/task/processor/processor_options.py

```diff
@@ -11,23 +11,69 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Options to configure Embedding APIs."""
+"""Options to configure Vision APIs."""
```

Comment:
Let's be even more generic here and go for "Options to configure processors"? We're going to have to change it anyway in the future with other types of APIs.